ci/lava/tests/test-avahi-discovery.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Script to run avahi discovery test."""
+
+from datetime import datetime
+import time
+
+
+class TestRunner:
+    """Class to encapsulate the avahi discover of a DUT."""
+
+    def _test_avahi_browse(self, execute_helper):
```

Comment:
Can we have this method at the end of the class?

Reply:
Can we also rename it to something else? (_run_avahi_browse, _discover_ssh_service ?)

Reply:
renamed to _run_avahi_browse and moved to the end of the class.